TestModels/SimpleTypes/SimpleBlob/runtimes/rust/src/operation/get_blob/_get_blob_input.rs

```diff
@@ -0,0 +1,60 @@
+// Code generated by software.amazon.smithy.rust.codegen.smithy-rs. DO NOT EDIT.
+#[allow(missing_docs)] // documentation missing in model
+#[non_exhaustive]
+#[derive(::std::clone::Clone, ::std::cmp::PartialEq, ::std::fmt::Debug)]
+pub struct GetBlobInput {
+    #[allow(missing_docs)] // documentation missing in model
+    pub value: ::std::option::Option<::std::vec::Vec<::std::primitive::u8>>,
```

Comment:
Just FYI, I see that in practice `smithy-rs` uses `::aws_smithy_types::Blob` instead: https://github.com/smithy-lang/smithy-rs/blob/main/rust-runtime/aws-smithy-types/src/blob.rs

We do want to avoid AWS-specific types so I think using `Vec[u8]` is the right call, but I mention this in case there are compatibility concerns down the road.